cli/src/action/circuit/api.rs

```diff
@@ -239,6 +241,49 @@ pub struct CircuitSlice {
     pub roster: Roster,
 }
 
+impl fmt::Display for CircuitSlice {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut display_string = format!(
+            "Circuit: {}\n    Management Type: {}\n",
+            self.id, self.circuit_management_type
+        );
+
+        for member in self.members.iter() {
+            display_string += &format!("\n    {}\n", member);
+            for service in self.roster.iter() {
+                if service.allowed_nodes().contains(member) {
+                    display_string += &format!(
+                        "        {} ({}): {}\n ",
+                        "Service",
```

Comment:
This can just be part of the format string

Reply:
Fixed